test/http-test.js

```diff
@@ -454,6 +454,60 @@ describe('HTTP', function() {
     assert.strictEqual(blocks.length, 10);
   });
 
+  // depends on the previous test to generate blocks
```

Comment:
Looks like existing tests need to be updated to generate an initial state in a before hook. Also looking more at the `http-test.js` and could probably be separated similar to `node-rpc-test.js` and `wallet-rpc-test.js` has been, it doesn't look like `/block/${hash}` is tested, aside from the additional `getBlock` call added below.

Reply:
Something to look at in another PR.

Reply:
Also, nit formatting of the comment (minor).